cassandra-4/pom.xml

```diff
@@ -265,6 +265,21 @@
                                     <tarLongFileMode>posix</tarLongFileMode>
                                 </configuration>
                             </execution>
+                            <execution>
```

Comment:
@keriharris @smiklosovic 
How about using `maven-shade-plugin`? The fatjar can be created with it but it provides an option to exclude a set of dependencies. That way we can get both fatjar with the necessary dependencies and the `tar.gz` as well. WDYT?

Reply:
@jpechane worth to investigate and compare the results!